test/run_pass/require_add.js

```diff
@@ -0,0 +1,24 @@
+/* Copyright 2015 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+  @NOTEST
+*/
+
+var x = require("buffer");
+
+exports.add = function(a, b) {
+  return a + b;
+};
```

Comment:
@ILyoan Yes, I just copy it. It there any way to add a test in subdirectory into our test set ?